Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1,9 +1,117 @@
+<h1>
+    Hinweise zur Beendigung der Verarbeitung personenbezogener
+</h1>
+<p>
+    ACHTUNG: Die nachfolgenden Hinweise gelten ab dem 1. Mai 2023. Die bis zum 30. April 2023
+    gültige Datenschutzerklärung ist unten angefügt.
+    Alle bisherigen Versionen der Datenschutzerklärung finden Sie unter: <a>https://www.coronawarn.app/de/privacy</a>
+</p>
+<p>
+    <strong>1. Beendigung der Datenverarbeitung</strong>
+</p>
+<p>
+    Am 1. Mai 2023 geht die Corona-Warn-App in den Ruhemodus. In der Folge wird der bisherige
+    Funktionsumfang beschränkt. Die meisten Funktionen einschließlich der Warnfunktion und des
+    Testergebnisabrufs stehen dann nicht mehr zur Verfügung.
+    Die Datenverarbeitung auf dem Serversystem des Robert Koch-Instituts (RKI), die für diese
+    Funktionen erforderlich ist, wird mit Beginn des Ruhemodus beendet. Das RKI verarbeitet ab
+    diesem Zeitpunkt im Zusammenhang mit diesen Funktionen keine personenbezogenen Daten mehr. Die
+    auf dem Serversystem des RKI für diese Funktionen gespeicherten personenbezogenen Daten werden
+    vollständig gelöscht.
+    Auf die in der Corona-Warn-App gespeicherten COVID-Zertifikate und die Einträge im
+    Kontakt-Tagebuch haben Sie jedoch weiterhin Zugriff, solange die App auf Ihrem Smartphone
+    installiert ist. Sie können jedoch keine neuen COVID-Zertifikate in der App speichern und
+    erhaltenen keine Hinweise zur Erneuerung von COVID-Zertifikaten. Wenn Sie die gespeicherten
+    COVID-Zertifikate und das Kontakt-Tagebuch weiterhin verwenden wollen, beachten Sie bitte die
+    Hinweise am Ende dieses Abschnitts.
+    Mit Aktivierung des Ruhemodus wird die Corona-Warn-App das COVID-19-Benachrichtigungssystem
+    Ihres Betriebssystems, mit dem die Begegnungsdaten ausgesendet und Begegnungen aufgezeichnet
+    werden (iPhone: „Begegnungsmitteilungen“; Android-Smartphones: „COVID-19-Benachrichtigungen“)
+    deaktivieren. Sie können das COVID-19-Benachrichtigungssystem jederzeit auch selbst in den
+    Einstellungen Ihres Betriebssystems deaktivieren. Ihr Betriebssystem löscht die auf ihrem
+    Smartphone gespeicherten Begegnungsdaten nach 14 Tagen automatisch. Falls Sie eine frühere
```

Comment:
Typo:

"Ihr Betriebssystem löscht die auf ihrem Smartphone gespeicherten Begegnungsdaten nach 14 Tagen automatisch."

should be auf "Ihrem Smartphone"